ONSdigital/census-rh-service
# Cr 879 rh link endpoint

# Motivation and Context
Big change as the logic is very involved. See CR text for detailed pseudo code.

# How to test?
Run unit tests for automated testing.
See CR-879 comments for instructions on manually testing.

# Links
https://collaborate2.ons.gov.uk/jira/browse/CR-879
